Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
+    yum clean all && /usr/sbin/sshd-keygen
+
+RUN mkdir /var/run/sshd
+
+# TODO: is this needed? https://git.io/vrplM
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+# gems
+RUN ["bash", "-c", "gem install --no-ri --no-rdoc bunny"]
+
+# git user
+RUN groupadd -r git &&\
+     useradd -r -s /bin/bash -g git git && \
+     mkdir --parent /home/git && \
+     chown -R git:git /home/git
+
+# gitlab-shell setup
+COPY . /home/git/gitlab-shell
+WORKDIR /home/git/gitlab-shell
+RUN ["bash", "-c", "./bin/install"]
+
+COPY authorized_keys /home/git/.ssh/authorized_keys
+
+USER root
```

Comment:
Sure!

https://github.com/openshift/postgresql/blob/master/9.2/Dockerfile#L30
https://github.com/openshift/postgresql/blob/master/9.2/root/usr/share/container-scripts/postgresql/common.sh#L117-L126

https://github.com/openshift/s2i-python/blob/master/2.7/Dockerfile#L18
https://github.com/openshift/s2i-python/blob/master/2.7/contrib/etc/generate_container_user


I think what we need is to install nss_wrapper is not installed, then export some envs on runtime to make use of it.